templates/components/common/navigation-menu.html

```diff
@@ -2,15 +2,26 @@
     <div class="navPages-quickSearch">
         {{> components/common/quick-search}}
     </div>
-    <ul class="navPages-list">
+    <ul class="navPages-list{{#if theme_settings.menu_display_depth_max}} navPages-list-depth-max{{/if}}">
```

Comment:
it's been changed - navigation_design can be 'simple' or 'alternate' where 'simple' is the default, existing design.